issuing/client_cardholders_test.go

```diff
@@ -0,0 +1,437 @@
+package issuing
+
+import (
+	"github.com/checkout/checkout-sdk-go/common"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+
+	"github.com/checkout/checkout-sdk-go/configuration"
+	"github.com/checkout/checkout-sdk-go/errors"
+	"github.com/checkout/checkout-sdk-go/mocks"
+)
+
+var (
+	createdDate      = time.Now()
+	lastModifiedDate = time.Now().Add(-5 * time.Hour)
+	phone            = &common.Phone{
+		CountryCode: "+1",
+		Number:      "415 555 2671",
+	}
+	address = &common.Address{
+		AddressLine1: "Checkout.com",
+		AddressLine2: "90 Tottenham Court Road",
+		City:         "London",
+		State:        "London",
+		Zip:          "W1T 4TJ",
+		Country:      "GB",
+	}
+	document = &CardholderDocument{
+		Type:            "national_identity_card",
+		FrontDocumentId: "file_6lbss42ezvoufcb2beo76rvwly",
+		BackDocumentId:  "file_aaz5pemp6326zbuvevp6qroqu4",
+	}
+	links = map[string]common.Link{
+		"self": {
+			HRef: &[]string{"https://api.checkout.com/issuing/cardholders/crh_d3ozhf43pcq2xbldn2g45qnb44"}[0],
+		},
+		"cards": {
+			HRef: &[]string{"https://api.checkout.com/issuing/cards"}[0],
+		},
+	}
+)
+
+func TestCreateCardholder(t *testing.T) {
+	var (
+		response = CardholderResponse{
+			HttpMetadata: common.HttpMetadata{
+				Status:     "Created",
+				StatusCode: 201,
```

Comment:
Please use the structs that are in `mocks/metadata_mocks` for this

Reply:
👍🏽